airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json

```diff
@@ -0,0 +1,362 @@
+              }
+            }
+          }
+        },
+        "errors": {
+          "type": ["null", "array"],
+          "items": {
+            "type": ["null", "string", "object"],
+            "properties": {
+              "flat_amount": {
+                "type": ["null", "integer"]
+              },
+              "unit_amount": {
+                "type": ["null", "integer"]
+              },
+              "up_to": {
+                "type": ["null", "integer"]
+              }
+            }
+          }
+        },
+        "eventually_due": {
+          "type": ["null", "array"],
+          "items": {
+            "type": ["null", "string", "object"],
+            "properties": {
+              "flat_amount": {
+                "type": ["null", "integer"]
+              },
+              "unit_amount": {
+                "type": ["null", "integer"]
+              },
+              "up_to": {
+                "type": ["null", "integer"]
+              }
+            }
+          }
+        },
+        "past_due": {
+          "type": ["null", "array"],
+          "items": {
+            "type": ["null", "string", "object"],
+            "properties": {
+              "flat_amount": {
+                "type": ["null", "integer"]
+              },
+              "unit_amount": {
+                "type": ["null", "integer"]
+              },
+              "up_to": {
+                "type": ["null", "integer"]
+              }
+            }
+          }
+        },
+        "pending_verification": {
+          "type": ["null", "array"],
+          "items": {
+            "type": ["null", "string", "object"],
+            "properties": {
+              "flat_amount": {
+                "type": ["null", "integer"]
+              },
+              "unit_amount": {
+                "type": ["null", "integer"]
+              },
+              "up_to": {
+                "type": ["null", "integer"]
+              }
+            }
+          }
+        }
+      }
+    },
+    "id_number_provided": {
+      "type": ["null", "boolean"]
+    },
+    "last_name": {
+      "type": ["null", "string"]
+    },
+    "metadata": {
+      "type": ["null", "object"],
+      "properties": {
+        "id_number_provided": {
+          "type": ["null", "boolean"]
+        }
+      }
+    },
+    "relationship": {
+      "type": ["null", "object"],
+      "properties": {
+        "director": {
+          "type": ["null", "boolean"]
+        },
+        "executive": {
+          "type": ["null", "boolean"]
+        },
+        "owner": {
+          "type": ["null", "boolean"]
+        },
+        "percent_ownership": {
+          "type": ["null", "string"]
+        },
+        "representative": {
+          "type": ["null", "boolean"]
+        },
+        "title": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "requirements": {
+      "type": ["null", "object"],
+      "properties": {
+        "alternatives": {
```

Comment:
The properties of this object doesn't align with what stripe says the structure should be. https://stripe.com/docs/api/persons/object#person_object-requirements-alternatives

Reply:
According to the docs, `alternatives` should have 2 properties `alternative_fields_due` and `original_fields_due` that are arrays of strings.